GaloyMoney/galoy
# fix: yarn lock update issues

Fix yarn lock issues. We need to find a way to avoid this, it is probably a merge issue.

This error was found after rebase https://github.com/GaloyMoney/galoy/pull/1178 (it is throwing some errors only updating yarn.lock)

How to test:
- checkout main branch
- remove yarn.lock and node_modules
- run `yarn install`
- try to run `make check-code` or unit tests
